packages/select-rich/test/lion-select-rich.test.js

```diff
@@ -189,6 +189,36 @@ describe('lion-select-rich', () => {
       // @ts-ignore allow protected access in tests
       expect(el._invokerNode.shadowRoot.firstElementChild.textContent).to.equal('30');
     });
+
+    it('inherits the content width including arrow width', async () => {
+      const arrowWidth = 30;
+      const el = /** @type {LionSelectRich} */ (await fixture(html`
+        <lion-select-rich arrow-width=${arrowWidth}>
+          <lion-option .choiceValue=${10}>Item 1</lion-option>
+          <lion-option .choiceValue=${10}>Item 2 with long label</lion-option>
+        </lion-select-rich>
+      `));
+      el.opened = true;
+      const options = el.formElements;
+      await aTimeout(0);
+      expect(el._invokerNode.clientWidth).to.equal(options[1].clientWidth + arrowWidth);
+
+      const newOption = /** @type {LionOption} */ (document.createElement('lion-option'));
+      newOption.choiceValue = 30;
+      newOption.textContent = '30 with longer label';
+
+      el._inputNode.appendChild(newOption);
+      el.requestUpdate();
```

Comment:
I believe this is not needed, because align invoker width happens as a consequence of add/removeFormElement, which happens if you append a child option that registers to the select rich.